source/releasenotes/2025-03-19-new-relic-ip-ranges.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "New Relic Synthetics IP Ranges – Update Your Allowlist"
+published_date: "2025-03-19"
+categories: [tools-apis, action-required]
+---
+New Relic is updating the IP ranges used for Synthetics monitoring. Most Pantheon customers do not need to take action. However, if you have an allowlist set up for New Relic Synthetics traffic (in either [Advanced Global CDN](/guides/agcdn) or another edge network provider), you may need to update your configurations before **April 14, 2025**, to ensure uninterrupted monitoring.
```

Comment:
```suggestion
New Relic is updating the IP ranges used for Synthetic Monitoring. Most Pantheon customers do not need to take action. However, if you have an allowlist set up for New Relic Synthetics traffic (in either [Advanced Global CDN](/guides/agcdn) or another edge network provider), you may need to update your configurations before **April 14, 2025**, to ensure uninterrupted monitoring.
```

"New Relic Synthetics" is the feature, but it's still referred to "Synthetic Monitoring" at least as far as I can grok from https://newrelic.com/lp/synthetics